src/DynamoCoreWpf/Views/PackageManager/PackagePathView.xaml

```diff
@@ -1,242 +1,141 @@
+                                <Binding Path="DataContext.RootLocations" RelativeSource="{RelativeSource AncestorType={x:Type UserControl}}"/>
+                                <!-- TODO when implementing toggles - one approach is to create these two properties on PackagePathViewModel and uncomment this.
+                                <Binding Path="DataContext.StdLibPathDisabled" RelativeSource="{RelativeSource AncestorType={x:Type UserControl}}"/>
+                                <Binding Path="DataContext.CustomPathsDisabled" RelativeSource="{RelativeSource AncestorType={x:Type UserControl}}"/>
+                                -->
+                            </MultiBinding>
+                        </DataTrigger.Binding>
+                        <Setter TargetName="PathTextBlock" Property="Opacity" Value=".5" />
+                    </DataTrigger>
+                </DataTemplate.Triggers>
+            </DataTemplate>
         </ResourceDictionary>
-    </Window.Resources>
-
-    <Grid Margin="10"
-          TextBlock.Foreground="#cccccc"
-          TextBlock.FontSize="14"
-          TextBlock.FontFamily="{StaticResource OpenSansRegular}">
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto"></RowDefinition>
-            <RowDefinition Height="auto"></RowDefinition>
-            <RowDefinition Height="*"></RowDefinition>
-            <RowDefinition Height="auto"></RowDefinition>
-        </Grid.RowDefinitions>
-        <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="auto"></ColumnDefinition>
-            <ColumnDefinition Width="*"></ColumnDefinition>
-        </Grid.ColumnDefinitions>
-
-        <StackPanel Orientation="Vertical"
-                    Grid.Row="0"
-                    Grid.Column="0"
-                    Grid.ColumnSpan="2">
-            <TextBlock TextWrapping="Wrap"
-                       Text="{x:Static p:Resources.PackagePathViewSummary1}" />
-            <TextBlock TextWrapping="Wrap"
-                       FontStyle="Italic"
-                       Text="{x:Static p:Resources.PackagePathViewSummary2}" />
-            <TextBlock TextWrapping="Wrap"
-                       FontStyle="Italic"
-                       Text="{x:Static p:Resources.PackagePathViewSummary3}" />
-        </StackPanel>
-
-        <TextBlock Grid.Row="1"
-                   Grid.Column="1"
-                   Margin="0,10,0,5"
-                   Text="{x:Static p:Resources.PackagePathViewHeading}" />
+    </UserControl.Resources>
+    <StackPanel Orientation="Vertical" Margin="0,6,0,0">
+        <DockPanel>
+            <Label  Content="{x:Static p:Resources.PackagePathPreferencesTitle}" 
+                                            Padding="5,5,5,5"
+                                            VerticalAlignment="Center"
+                                            HorizontalAlignment="Left"
+                                            FontSize="14"
```

Comment:
same as above ...

Reply:
added a new style for textblocks with the font size and text color.